A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_BLEED.js

```diff
@@ -79,6 +84,135 @@ var A320_Neo_LowerECAM_BLEED;
+
+            //find xbleed switch state
+            var currentXbleedState = SimVar.GetSimVarValue("L:A32NX_KNOB_OVHD_AIRCOND_XBLEED_Position", "Position(0-2)")
+
+            if(currentXbleedState == 2 ){
+                this.xBleed[0].setAttribute("visibility", "visible")
+                this.xBleed[1].setAttribute("visibility", "hidden")
+            } else if(currentApuBleedSate && currentXbleedState == 1){
+                this.xBleed[0].setAttribute("visibility", "visible")
+                this.xBleed[1].setAttribute("visibility", "hidden")
+            } else {
+                this.xBleed[0].setAttribute("visibility", "hidden")
+                this.xBleed[1].setAttribute("visibility", "visible")
+            }
+
+            //find ram air state
+            var currentRamState = SimVar.GetSimVarValue("L:A32NX_AIRCOND_RAMAIR_TOGGLE", "bool")
+
+            if(currentRamState) {
+                this.ramAir[0].setAttribute("visibility", "hidden")
+                this.ramAir[1].setAttribute("visibility", "visible")
+            } else {
+                this.ramAir[0].setAttribute("visibility", "visible")
+                this.ramAir[1].setAttribute("visibility", "hidden")
+            }
+
+             //find pack flow state
+             var currentPackFlow = SimVar.GetSimVarValue("L:A32NX_KNOB_OVHD_AIRCOND_PACKFLOW_Position", "Position(0-2)")
+
+            if(currentPackFlow == 0){
+                 this.packFlow[0].setAttribute("visibility", "visible")
+                 this.packFlow[1].setAttribute("visibility", "hidden")
+                 this.packFlow[2].setAttribute("visibility", "hidden")
+            } else if(currentPackFlow == 1) {
+                 this.packFlow[0].setAttribute("visibility", "hidden")
+                 this.packFlow[1].setAttribute("visibility", "visible")
+                 this.packFlow[2].setAttribute("visibility", "hidden")
+            } else {
+                 this.packFlow[0].setAttribute("visibility", "hidden")
+                 this.packFlow[1].setAttribute("visibility", "hidden")
+                 this.packFlow[2].setAttribute("visibility", "visible")
+            }
+
+            //mock logic for the bleed page temperatures and pressures, to be replaced/updated/removed when the cond-packs system is implemented
+
+            var eng1Running = SimVar.GetSimVarValue("ENG COMBUSTION:1", "bool")
+            var eng2Running = SimVar.GetSimVarValue("ENG COMBUSTION:2", "bool")
+
+            var eng1TMP = SimVar.GetSimVarValue("ENG EXHAUST GAS TEMPERATURE:1", "Rankine")      
+            var eng1PSI = SimVar.GetSimVarValue("TURB ENG BLEED AIR:1", "Ratio (0-16384)")
+
+            var eng2TMP = SimVar.GetSimVarValue("ENG EXHAUST GAS TEMPERATURE:2", "Rankine")
+            var eng2PSI = SimVar.GetSimVarValue("TURB ENG BLEED AIR:2", "Ratio (0-16384)")
+
+            var xBleedValveOpen = false
+
+            if((currentXbleedState == 2 || (currentXbleedState == 1 && currentApuBleedSate))){
```

Comment:
Replace with `if(currentXbleedState == 2 || (currentXbleedState == 1 && currentApuBleedSate)){`

You had an extra pair of brackets which werent needed